Gradle is not happy about Unicode comments

@rsninja722 seems to use a different locale than the rest of us. This causes this error and others in files he has worked in. Can someone clean it all up? 
```
/github/workspace/src/main/java/frc/lib5k/components/limelight/Limelight.java:143: error: unmappable character (0x99) for encoding US-ASCII
     * Sets limelight???s streaming mode
```

Here is a link to a CI build log with a list of every time this error has occured:

https://hastebin.com/raw/zonewesiqo

The errors are reported at the very bottom of this file

--- a/src/main/java/frc/lib5k/components/limelight/Limelight.java
+++ b/src/main/java/frc/lib5k/components/limelight/Limelight.java
@@ -110,7 +110,7 @@ public class Limelight {
     }
 
     /**
-     * The pipeline’s latency contribution
+     * The pipeline's latency contribution
      * 
      * @return ms
      */
@@ -119,28 +119,28 @@ public class Limelight {
     }
 
     /**
-     * Sets limelight’s LED state
+     * Sets limelight's LED state
      */
     public void setLEDMode(LEDMode mode) {
         SLED.setNumber(mode.getVal());
     }
 
     /**
-     * Sets limelight’s operation mode
+     * Sets limelight's operation mode
      */
     public void setCameraMode(CameraMode mode) {
         cameraMode.setNumber(mode.getVal());
     }
 
     /**
-     * Sets limelight’s current pipeline
+     * Sets limelight's current pipeline
      */
     public void setPipelineID(int id) {
         setPipeId.setNumber(id);
     }
 
     /**
-     * Sets limelight’s streaming mode
+     * Sets limelight's streaming mode
      */
     public void setStreamMode(int mode) {
         setStream.setNumber(mode);

